brands/service.go

```diff
@@ -250,7 +250,24 @@ func (s *brandServiceImpl) openDB() error {
 
 func (s *brandServiceImpl) reloadDB() error {
 	s.setDataLoaded(false)
-	return s.loadDB()
+
+	err := s.loadDB()
+	if err != nil {
+		log.Errorf("Error while creating BrandService: [%v]", err.Error())
+	}
+	var bBrands []berthaBrand
```

Comment:
If you can't load the DB file you still call TME and Bertha and also attempt to write that data to the DB/file.